Minio/Regions.cs

```diff
@@ -29,14 +29,22 @@ private Regions()
         /// <returns>Region corresponding to the endpoint. Default is 'us-east-1'</returns>
         internal static string GetRegion(string endpoint)
         {
-            if (endpoint.EndsWith("s3 - ap - northeast - 1.amazonaws.com"))
+            if (endpoint.EndsWith("s3-ap-northeast-1.amazonaws.com"))
```

Comment:
Can't we get the region from the endpoint string without having to use EndsWith method to pattern match individual regions?

Reply:
yes, updated PR.